tool/gravity/cli/status.go

```diff
@@ -88,6 +88,33 @@ func status(env *localenv.LocalEnvironment, printOptions printOptions) error {
 	return trace.Wrap(printStatus(operator, clusterStatus, printOptions))
 }
 
+func tailStatus(env *localenv.LocalEnvironment, operationID string) error {
+	if operationID == "" {
+		return trace.BadParameter("operation ID is required to tail operation logs")
+	}
+
+	operator, err := env.SiteOperator()
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	status, err := acquireClusterStatus(context.TODO(), env, operator, operationID)
+	if err == nil {
+		err = tailOperationLogs(operator, status.Operation.Key())
```

Comment:
Actually, during one of recent refactorings (I think it was expand-fsm) I made changes so the `Operation` field in the collected status object refers to the _last completed_ operation. There is now also `ActiveOperations` field on the status object which contains the operation(-s) that is currently in progress. So you probably want to use that one here. Also, there may be several operations in progress at the same time (joins), so in this case we should probably fail and ask to provide a specified operation-id flag to the command.

Also, as an idea, if there are no operations in progress, we could just fall back to this `Operation` field to display logs for the last completed operation, would be pretty convenient.

Reply:
I think it depends on whether the `operationID` parameter specifies a valid operation ID, in which case it will always query info for the specified operation.